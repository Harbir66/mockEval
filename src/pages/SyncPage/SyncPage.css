.sync-page {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  min-height: 100vh;
  min-width: 700px;
}

@media (max-width: 455px) {
  .sync-page {
    min-width: 455px;
  }
}
