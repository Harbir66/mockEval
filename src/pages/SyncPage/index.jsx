import React from 'react';
import './SyncPage.css';
import { NavBar, SyncBody } from '../../components';

function SyncPage() {
  return (
    <div className="sync-page">
      <NavBar />
      <SyncBody />
    </div>
  );
}

export default SyncPage;
