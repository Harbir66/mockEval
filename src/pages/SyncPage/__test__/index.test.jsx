import { render } from '@testing-library/react';
import React from 'react';
import SyncPage from '../index';

jest.mock('react-router-dom', () => ({
  useNavigate: jest.fn(),
}));

describe('SyncPage', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<SyncPage />);

    expect(asFragment).toMatchSnapshot();
  });
});
