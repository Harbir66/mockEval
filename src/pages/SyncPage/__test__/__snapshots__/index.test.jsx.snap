// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SyncPage should render correctly 1`] = `[Function]`;
